libs/core/src/scss/base/_variables.scss

```diff
@@ -87,19 +87,13 @@ $border-radius-round: 999px !default;
 /* Breakpoints
 ****************************************************************************/
 $breakpoints: (
-  small: 321px,
-  medium: 721px,
-  large: 1025px,
-  xlarge: 60em,
-  xxlarge: 80em,
+  xsmall: 0,
```

Comment:
can be removed and mapped locally in grid if needed at all.
We dont want a breakpoint that does "nothing"